libs/layout/src/lib/layout.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { OrchestratorCoreModule } from '@orchestrator/core';
+
+import { LayoutFlatComponent } from './layout-flat/layout-flat.component';
+import { FlexWrapDirective, FlexDirective, FlexDirectionDirective } from './flex';
+
+@NgModule({
+  imports: [CommonModule, FlexLayoutModule, OrchestratorCoreModule],
+  declarations: [LayoutFlatComponent, FlexWrapDirective, FlexDirective, FlexDirectionDirective],
+  exports: [LayoutFlatComponent],
```

Comment:
Other flex directives can be also exported so we and users can reuse them somewhere else.